PractiseJune14/SingleLinkedList/Node.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseJune14.SingleLinkedList
+{
+    public class Node<T>
+    {
+        public T Value { get; set; }
+
+        public Node<T> Next { get; set; }
+        
+        public Node(T value)
+        {
+            if (value == null)
+                throw new ArgumentNullException();
```

Comment:
Please separate if statement with empty lines to make it more readable.

Reply:
fixed